src/main/java/core/basesyntax/LifeHackCalculus.java

```diff
@@ -0,0 +1,69 @@
+package core.basesyntax;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+
+public class LifeHackCalculus {
```

Comment:
```suggestion
public class ContentCalculations {
```
make class names more understandable. the best option would be if you combine these 2 classes and make separate methods for reading and writing

Reply:
1. Pushed commits were necessary for easy switching between devices. Also I had some git problems, but I'll keep it noted, thanks.
2. Ok, but due to "1 homework - 1 PR" practice student can describe changes only once per naming PR, all other changes can be described in commits.
3. Could you be more specific about part of logic I need to implement in separate method?
4. Namestyling for the class recommendation accepted.